services/api-server/src/simcore_service_api_server/models/schemas/profiles.py

```diff
@@ -28,6 +29,7 @@ class UserRoleEnum(StrAutoEnum):
 
 
 class Profile(ProfileCommon):
+    id_: IdInt = Field(alias="id")
```

Comment:
MINOR: there is a `UserID` type in place. Please use thatone

Reply:
Thanks, I changed `IdInt` -> `UserID` in the schema for the webserver as well. They are both aliases for `PositiveInt`, so that should be safe